### PR TITLE
Added OpenGraph support for public articles

### LINKED ISSUE #2289: Add information in public articles
### Issue details

When we share an article (http://v2.wallabag.org/share/57d954ef5e8745.84078008), we need to display some data (for example: "This article has been shared with wallabag. Use wallabag now!")

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | #2289 
| License       | MIT

Fix #2289

@tcitworld @j0k3r tell me if we need more OpenGraph tags.

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.da.yml [hunk 1]
```diff
@@ -208,6 +208,8 @@ entry:
         url_label: 'Url'
         # is_public_label: 'Public'
         save_label: 'Gem'
+    # public:
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.da.yml]: The public line can be uncommented in all translation files

File: src/Wallabag/CoreBundle/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -208,6 +208,8 @@ entry:
         url_label: 'Url'
         is_public_label: 'Public'
         save_label: 'Save'
+    public:
+        shared_by_wallabag: "This article has been shared by <a href='https://www.wallabag.org'>wallabag</a>"
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: Should we instead display the url of the current instance or not?

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: why not.

File: src/Wallabag/CoreBundle/Resources/translations/messages.da.yml (hunk 1 above)
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/translations/messages.da.yml]: will change that.

File: src/Wallabag/CoreBundle/Resources/translations/messages.en.yml (hunk 2 above)
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: It's an open question I don't know if it's better to put a link to the _brand_ or to the user instance.

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: the user instance could be a good option.

File: src/Wallabag/CoreBundle/Resources/views/themes/common/Entry/share.html.twig [hunk 3]
```diff
@@ -27,11 +27,18 @@
                 width: 600px;
             }
         </style>
+        <meta property="og:title" content="{{ entry.title | raw }}" />
+        <meta property="og:type" content="article" />
+        <meta property="og:url" content="{{ app.request.uri }}" />
+        {% if entry.previewPicture is not null %}
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/views/themes/common/Entry/share.html.twig]: @nicosomb Maybe if the previewPicture is null we could have displayed wallabag's logo ?

**Pull request merged**